Typescript compilation error due to missing Declaration file for test utils

Remember, an issue is not the place to ask questions. You can use [Stack Overflow](http://stackoverflow.com/questions/tagged/ibm-watson) for that, or you may want to start a discussion on the [dW Answers](https://developer.ibm.com/answers/questions/ask/?topics=watson).

 Before you open an issue, please check if a similar issue already exists or has been closed before.

**Check service status**
N/A

**Overview**
_What is the issue? Be concise and clear._
Whe


**Expected behavior**
Compiles

**Actual behavior**
Throws compilation error

```
node_modules/ibm-cloud-sdk-core/index.d.ts:25:15 - error TS7016: Could not find a declaration file for module './test/utils'. '.../node_modules/ibm-cloud-sdk-core/test/utils/index.js' implicitly has an 'any' type.

25 export * from './test/utils';
                 ~~~~~~~~~~~~~~

Found 1 error.
``` 

**How to reproduce**
Create a typescript project that uses ibm-watson and attempt to compile

**Screenshots**
N/A

**SDK Version**
5.1.0

**Additional information:**
- OS: macOS
- Which version of `Node` are you using?: v10.16.3

**Additional context**
N/A

--- a/package.json
+++ b/package.json
@@ -88,7 +88,7 @@
     "axios": "^0.18.0",
     "camelcase": "^5.3.1",
     "extend": "~3.0.2",
-    "ibm-cloud-sdk-core": "^1.0.0",
+    "ibm-cloud-sdk-core": "^1.3.1",
     "isstream": "~0.1.2",
     "websocket": "^1.0.28"
   },

